src/main/java/ag/boersego/bgjs/BGJSGLView.kt

```diff
@@ -106,16 +114,18 @@ open class BGJSGLView(engine: V8Engine, val textureView: V8TextureView) : JNIV8O
             callbacksForThisRedraw.addAll(callbacksRedraw)
```

Comment:
Should not be used here; see explanation provided in  messenger;
we should have something like "fun requestRedraw" that we could call to tell the javascript layer to reqeust a new animation frame and render should we ever need to reinitialize the opengl context for some reason other than resize.